experimental/framework/frs-client/src/test/CreateClient.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { generateUser } from "@fluidframework/server-services-client";
+import {
+    FrsClient,
+    FrsConnectionConfig,
+    InsecureTokenProvider,
+} from "..";
+
+export class CreateClient {
+    // use FrsClient remote mode will run against live Frs service. Default to running Tinylicious for PR validation
+    // and local testing so it's not hindered by service availability
+    private readonly useFrs: boolean;
+    private readonly user: any;
+    private readonly tenantKey: string;
+    private readonly connectionConfig: FrsConnectionConfig;
+
+    constructor() {
+        this.useFrs = process.env.FLUID_CLIENT === "frs";
```

Comment:
Usage of environment variables suggests that
1. This is only for node environment (i.e. this is not isomorphic code).
2. This is likely not for production environments (i.e. it is supposed to be used in testing only).
Does this match reality?
If not, is this really what we want?
Usage of environment variables is similar to usage of globals / singletons, even worse, so we should not use it in code that is supposed to scale.

Reply:
since we need to toggle the tests to run between remote vs local mode based on context, the only alternative that I see that doesn't involve the use of environment variable is having separate test suites, one for each respective Frs modes.